versions/3.0.4.md

```diff
@@ -249,12 +249,24 @@ This is the root object of the [OpenAPI document](#openapi-description).
 | <a name="oas-components"></a>components     |            [Components Object](#components-object)             | An element to hold various schemas for the document.                                                                                                                                                                                                                                                                                                                                                      |
-| <a name="oas-security"></a>security         |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array. |
+| <a name="oas-security"></a>security         |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. See [The `security` Field](#the-security-field).                                                                                                                                                                                                                                                                                  |
 | <a name="oas-tags"></a>tags                 |                  [[Tag Object](#tag-object)]                   | A list of tags used by the document with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operation-object) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.                                 |
 | <a name="oas-external-docs"></a>externalDocs | [External Documentation Object](#external-documentation-object) | Additional external documentation.                                                                                                                                                                                                                                                                                                                                                                        |
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+###### The `security` Field
+
+The `security` field describes how requests are authorized:
+
+- If omitted, then nothing can be inferred about the authorization requirements; the behaviour is implementation-defined.
+- If present but empty (`security: []`), then the behaviour is undefined.
```

Comment:
I think this is implementation-defined rather than undefined.  There are only two viable options (which should be stated explicitly): allow-all or deny-all.  It is implementation-defined as to which one of those two will happen.  Undefined means that arbitrary (and arbitrarily wrong) behavior can happen, and I don't think that's a concern here.  (Implementation-defined and undefined are terms that are defined within the OAS for 3.0.4 and 3.1.1, for anyone reading who was not aware of that)